### DISCUSSION TITLE
user-guide/plugins/style

### DISCUSSION THREAD

**Discussion by giscus** [zotero-chinese/wiki#154] [文档评论区]
# user-guide/plugins/style

Ethereal Style for Zotero 关于插件 这是一个多彩的插件，用于增强 Zotero 的界面功能，让您享受阅读文献的乐趣！ 项目主页：https://github.com/MuiseDestiny/zotero-style 下载地址：插件商店 - Zotero 中文社区 搜索 style。 首先需要明确的是，本插件适用人群：想美化自...

https://main--zotero-zh.netlify.app/user-guide/plugins/style.html

**Comment by shanleyao** [comment]
期刊标签怎么显示 sciTop期刊？
easyScholar  sciUpTop	中科院升级版Top分区

**Reply by Chikit-L** [reply]
首先需要现在es里面添加对应的数据集，然后在期刊表标签的设置里面添加对应的简称就可以。以“中科院升级版Top分区-2023年12月”数据集为例，在es里面的简称是“TOP”，添加数据集以后，在期刊标签的列设置添加TOP，设置完成后重新刷新期刊标签即可。

**Comment by littleBillGates** [comment]
使用快捷键“shift+P”进入“标注”之后，在添加标注方案这个页面，只有加号没有减号，我不小心创建了很多标注方案，怎么删除多余的标注方案呢？？？

**Reply by Chikit-L** [reply]
鼠标放在标注的颜色区域，长按鼠标右键

**Reply by littleBillGates** [reply]
感谢感谢

**Comment by ARLONGJING** [comment]
已经开启了  PDF Style功能，但是在PDF的阅读界面，上方没有改背景颜色的选项。

**Reply by Chikit-L** [reply]
检查一下插件版本是不是4.2.1

**Comment by blackbearha** [comment]
点了全文翻译没有任何反馈，该怎么解决呢

**Reply by Chikit-L** [reply]
只能等修复

**Reply by HangChi** [reply]
现在还可以全文翻译吗，为什么我看不到全文翻译的按钮

**Reply by Chikit-L** [reply]
> 现在还可以全文翻译吗，为什么我看不到全文翻译的按钮

适配7的style最新版是有的

**Comment by tolvaptan6** [comment]
那个加粗和不加粗的已读/未读状态我在最新版的插件里面没找到，是插件冲突了还是说这个功能取消了？我感觉这个功能很实用

**Reply by Chikit-L** [reply]
看起来是没有了

**Comment by iceee127** [comment]
7的全文翻译有些PDF文档无法使用怎么办呀？如果现在退回6的话之前的全文翻译还能用吗？o(╥﹏╥)o

**Reply by Chikit-L** [reply]
无法翻译的问题请反馈给作者。6的全文翻译的通道已经关闭。

**Comment by Siki-cloud** [comment]
超赞的项目！顺便问问~有希望自动导入CCFA等级吗？

**Reply by Chikit-L** [reply]
期刊标签吗？这应该是easyscholar的部分功能。可以查看一下ec的自定义数据集里面有没有这个数据集。

**Comment by Huxinyuu** [comment]
zotero6和style2.6.7为什么右击列了之后没有“状态”和“简记”可以选择啊？谢谢！

**Reply by Chikit-L** [reply]
没有的话就是不支持，很多功能都是在Z7上才有，并且随着版本的更新，有一些功能也被砍掉了。

**Comment by chenxiaojie92** [comment]
添加多个试图组需要付费吗

**Reply by Chikit-L** [reply]
不需要，免费版就可以

**Comment by alicebear1203** [comment]
我的Zotero7，右鍵點選列，沒有出現列設置，這樣應該不大正常？

**Reply by Chikit-L** [reply]
有一些列沒有列設置是正常的，如果所有的列都沒有列設置那就是存在問題（例如 標題列）

**Reply by alicebear1203** [reply]
我標題列真的也沒有列設置欸！想請問知道大概發生什麼問題嗎？希望不要要重新安裝...
謝謝大神熱心回覆！

**Reply by Chikit-L** [reply]
> 我標題列真的也沒有列設置欸！想請問知道大概發生什麼問題嗎？希望不要要重新安裝... 謝謝大神熱心回覆！

標題列是會有列設置的，如果沒有的話，那就是插件存在問題，建議從以下幾個方面進行排查：檢查Zotero是否為最新版；檢查Style插件是否為最新版；禁用其他所有插件，排查是否是由於插件衝突所引起。如果以上方法都不行，那可以嘗試重新安裝Zotero（只要你的安裝路徑沒有發生改變，你的數據文件夾也不會受到影響，那就不會引起數據的丟失。

**Reply by jiaojiaodubai** [reply]
我遇到過有用戶在同一臺電腦上使用兩個顯示器導致無法顯示列設置，如果你屬於這種情況，可以嘗試先斷開其中一個屏幕做好設置

**Reply by alicebear1203** [reply]
@Chikit-L  好的，我等等來檢查看看。謝謝還特別說明我的數據不會丟失。
@jiaojiaodubai  我是用Mac筆電外接兩個螢幕，剛剛試著拔了一個螢幕或者回去用筆電螢幕，還是沒有出現，可能要再試試看上面大大提供的方法，不過還是很謝謝你！這感覺也能夠幫助到其他人！

**Reply by jiaojiaodubai** [reply]
對於插件而言（特別是 Zotero 6 上面的插件），某些更改並不會立即生效，需要重啓 Zotero 引導插件重新加載。我的建議和 Chikit 差不多：
1. 檢查 Zotero 版本，即依次點擊主菜單”幫助—檢查更新“，如果有更新請更新。
2. 檢查 Style 插件版本，你可以直接到(插件商店)[https://zotero-chinese.com/plugins/]下載安裝最新版。
3. 拔掉一個熒幕，重新啓動 Zotero（注意對於 Mac 電腦而言，僅關閉程序窗口並不會退出程序，要在程序塢中右鍵點擊”退出“），看看在這種”一般的條件下“，是不是能顯示列標題。

**Comment by ARLONGJING** [comment]
对文献添加tag（利用Action&Tags）后，会立马改变文献在该分类的排列顺序，如何使其不改变呢？

**Reply by Chikit-L** [reply]
不要按照标签对文献进行排序

**Comment by tych3nn** [comment]
您好，我想请问下“阅读时间”在不同电脑上不能同步是为什么，可以解决这个问题吗？

**Comment by iceee127** [comment]
升级到7正式版后，之前在全文翻译中的高亮全部不见了怎么办？急急急！

**Comment by Yuanle11** [comment]
我的标题上可视化的阅读进度条突然全部消失了，请问是因为什么呢？应该如何调整呢？而且我右键单击标题，进入列设置，只有Color，Opacity和Translate，并没有其他选项，问题会出在哪呢？感谢！

**Comment by aaalu26** [comment]


**Reply by aaalu26** [reply]
提示的问题是：Style has been disabled due to security or stability issues.

**Reply by Chikit-L** [reply]
> 更新到zotero7之后style无法使用了，重装了好多次都不行。请教一下大家怎么解决啊T^T

是不是没升级到最新的插件版本

**Comment by WonderYAQI** [comment]
各位大佬们，我的zotero7中使用这个插件。右上角粉色的A（annotation-manage）按钮好像在某一天之后消失了，请问有遇到同样问题的吗

**Comment by WUT-Dingzhen** [comment]
设置评级时会自动生成一个对应⭐数量的标签，如何不让它生成这个标签

**Reply by Chikit-L** [reply]
> 设置评级时会自动生成一个对应⭐数量的标签，如何不让它生成这个标签

看来这个功能应该是依靠标签来实现的，如果是这样的，那就没有办法。具体的您可以在github寻求作者帮助。

**Comment by Ying-Zi66** [comment]
我shift+p没有阅读高能进度条，并且我不想要标题显示阅读进度，如何操作？强迫症都是白色，不想有其他颜色，很别扭

**Reply by Chikit-L** [reply]
> 我shift+p没有阅读高能进度条，并且我不想要标题显示阅读进度，如何操作？强迫症都是白色，不想有其他颜色，很别扭

在编辑-设置-style-function 取消勾选对应的功能

**Reply by Ying-Zi66** [reply]
十分好的回答，成果解决问题

**Reply by Ying-Zi66** [reply]
请问具体是哪个功能？中文对应英文没找到

**Reply by 1527687658** [reply]
在标题列设置里面把透明度拉到最低就可以了

**Comment by LBJCG** [comment]
zotero7 style配置好后，想让标签置于标题的后边。但是每次重启zotero后，标签列会跑到创建者的后边。这是否是一个未修复的bug。

**Reply by Chikit-L** [reply]
可以在github向作者进行反馈

**Comment by shuaichennn** [comment]
请问刊名的列默认设置是什么，手贱删掉了刊名都不显示了

**Reply by Chikit-L** [reply]
publicationTitle, proceedingsTitle, university, publisher

**Reply by shuaichennn** [reply]
感谢迅速回复！

**Comment by Polaris-F** [comment]
zotero 7  条目聚焦后的这个蓝色可以修改吗，有点太亮 了 没有 zotero6的看着舒服。
如图 https://i.imgur.com/VZsg0Ol.png

**Comment by ccShark4652** [comment]
打开命令框进入标注，如果我把title全部删掉，以后就不能再修改title名称了。还有就是我有两个标注，怎么在它们之间切换

**Reply by ccShark4652** [reply]
长按可以切换，但是我把title删掉之后，长按就无效了，重新创一个一样的标注就好了

**Comment by Ying-Zi66** [comment]
请问style插件访问时间，我打开pdf和条目，访问时间不变化，怎么搞？求助
有点文献甚至没有访问时间，我打开了pdf，我设置的pdf打开是知云

**Reply by Chikit-L** [reply]
插件只对内置PDF阅读器生效。

**Comment by SunboyLisp** [comment]
我出现了一个问题，突然阅读时间不显示了，我就把这个插件重新安装了一下，后来之前的阅读时间数据丢失了怎么要怎么办？它不会保存阅读时间的数据在电脑上吗

**Reply by Chikit-L** [reply]
看起来应该是插件出现问题了，建议重装插件。

**Comment by SunboyLisp** [comment]
我还出现了个问题，没有显示期刊标签，不知道为啥，今天突然不行了

**Comment by Ceciliashing** [comment]
我的关系图谱为什么点击以后不出来呢，只有zotero功能的图谱

**Comment by WUT-Dingzhen** [comment]
给条目添加了嵌套标签之后，如A/B/C，标签视图下可见该标签，切换到嵌套标签视图为空白，不显示任何内容，如何解决

**Comment by 717510997** [comment]
中文文献的进度不能显示呢

**Comment by Heatherrrr701** [comment]
zotero7更新后抓取的知网文献（包括caj转pdf后的）都无法使用标注，鼠标光标显示一条杠也无法选中。但资料库中6保存的文献都可进行标注，且重装后shift+p或shift+p+tab都无法打开命令框。

**Comment by phebeMai** [comment]
文中提到的GPT 插件实现 快速 AI 生成简记，为什么我shift＋p调出来没有显示那个AI概括生成简记（有GPT3.5）？求大佬解惑

**Reply by phebeMai** [reply]
还有就是引用数这一列是不是更新后取消了，我右键标题那一列里面找不到，只有一个知网引用数，而且也不显示内容

**Comment by le876** [comment]
请问现在可以修改epub的背景颜色嘛，我现在可以修改pdf的背景颜色但是同样方式修改epub文件则无现象

**Comment by xiaopu3678** [comment]
为什么高能进度条我换了知云翻译打开PDF之后就不同步了，然后换回zotero打开方式后依旧不同步高能进度条。也没有找到记录时间间隔设置选项啊，求助 （zotero7）

**Comment by YQX0** [comment]
期刊标签中的field中只有小写的ccf会有些期刊不显示ccf标签，但是使用大写的CCF同样会有一些期刊不显示，只能同时使用ccf和CCF吗

**Comment by Athena-Psyer** [comment]
今天打开zotero之前有的阅读时间和高能进度条突然就没有了，重新读又会出现，但是很烦，不能一眼看出哪些读过了

**Reply by Chikit-L** [reply]
有可能你在使用同步盘方式进行同步，出现了数据冲突。

**Comment by SisterHuan** [comment]
请问一下设置了标签、期刊、进度等栏目之后，可以正常显示和使用。但是过几天就又恢复到之前的状态了，要重新设置一下。这种情况要如何解决

**Reply by Chikit-L** [reply]
日常bug，github反馈给作者看看吧

**Comment by wjgwt** [comment]
style 的搜索引擎是什么？

**Reply by Chikit-L** [reply]
style没有搜索引擎

**Comment by syuuraku** [comment]
阅读记录功能怎么打开呢？我在设置里面找不到 `read status` ，只有 `status`。

**Comment by Taemiil** [comment]
为什么我下载的是zotero 7，对应下载的style也是7版本的，但是显示插件不兼容呢

**Comment by sym823808458** [comment]
我这里是新安装的zotero7.0.11，windows版本，我的Graph view显示的内容并没有显示我文献的内容，关系图谱显示的是style的graph view 是怎么回事

**Comment by lijt787** [comment]
期刊标签不显示英文文章分区、影响因子等信息，中文文章可以显示北核，是怎么回事？

**Reply by Chikit-L** [reply]
英文文章是否有条目，条目是否有doi数据，数据集是否设置正确，数据是否刷新

**Comment by swaiwai-ss** [comment]
请问，Angewandte Chemie这个期刊为什么不显示期刊标签？在easyscholar中查询期刊等级，显示sciQ1，影响因子是16，但是在zotero中更新却显示not found，请问这是为什么？

**Reply by swaiwai-ss** [reply]
![屏幕截图 2025-03-25 142810](https://github.com/user-attachments/assets/a39f5997-3619-406f-be4f-998867ce76b7)
![屏幕截图 2025-03-25 143008](https://github.com/user-attachments/assets/8bef0856-cfec-4ea9-bc77-f64e11135be5)
附图是zotero的显示和期刊查询结果，求大神指导！

**Reply by Chikit-L** [reply]
easyscholar中并没有Angewandte Chemie期刊，你所说的`显示sciQ1，影响因子是16`的是Angewandte Chemie-international Edition期刊

**Reply by swaiwai-ss** [reply]
！！！感谢大佬！把刊名改成Angewandte Chemie-international Edition，现在显示了！

**Reply by Sylvie-Chai** [reply]
很需要“笔记预览 （仅支持 Zotero 6）”支持Zotero 7
有希望未来更新中支持嘛qvq

**Comment by Sylvie-Chai** [comment]
很需要“笔记预览 （仅支持 Zotero 6）”支持Zotero 7
有希望未来更新中支持嘛qvq

p.s.上一条评论错楼了，不好意思

**Reply by Chikit-L** [reply]
可以到github发issue向作者提供建议。

**Reply by Sylvie-Chai** [reply]
谢谢回复 我以为您是作者抱歉

**Comment by zhidacaishu** [comment]
请问现在只有pro版本能更换pdf背景颜色吗，我使用基础版本右上角无法选择颜色

**Reply by Chikit-L** [reply]
与是否付费无关。

**Comment by 52IvesWu** [comment]
为什么每次重新打开Zotero，列设置的顺序都会变化？比如我喜欢把期刊标签列放在最前面，但是经常重新打开Zotero时期刊标签列会跑到后面去

**Comment by Chikit-L** [comment]
插件bug，请到github向作者反馈
________________________________

**Comment by DarthYZK** [comment]
全文翻译之后再次打开zotero，翻译的文件就不能再显示原文了，shift+P也找不到命令。

**Comment by shy777-1** [comment]
求解，#标签列，有时候打开软件就不见了，需要重新在列表选项中打开，怎么解决。

**Comment by wtq20230227** [comment]
我在视图组中添加一个视图后，没有办法再添加新的视图，新增视图一栏变为灰色，点击无反应。

**Comment by minmin902** [comment]
在全文翻译时显示HTTP POsT https:/kermitt2-grobid.hf.space/api/processFulltextDocument failed withstatus code 500: [BAD INPUT DATA] PDF to XML conversion failed with error code：1
该怎么操作呢？

**Comment by eryuewu** [comment]
请问要实现：父文件夹右边的数量是其下所有子孙条目的总数量（现在只显示父文件夹自己的条目数量），应该如何设置呢？网上搜索到说要设置collection data xxx，但当前版本没有这个设置入口，是取消了吗

**Reply by Chikit-L** [reply]
查看-显示子分类中的条目

**Comment by renjia955-boop** [comment]
想问问可以使用快捷键吗，每次添加标签都需要滑动点击标签才能添加感觉好麻烦，不知道是不是我没找到怎么使用快捷键

**Comment by assguard** [comment]
macos 自动更新到5.8.7后 pdf背景无法自定义了 找不到设置页面

**Comment by Lee-Eleven** [comment]
插件安装后，Fields和Map区域都按照中文社区的模板设置，密钥输入也无误，zotero和style都是最新版本，但期刊标签只能在部分期刊中显示，手动更新期刊标签也没用，请问这种情况应该解决啊？

**Reply by Chikit-L** [reply]
不是所有的期刊都有分区，建议确认所查询期刊在对应的分类中存在分区

**Comment by ldz0935** [comment]
在列设置中自定义了field和map，但是默认的5个rankcolor只能显示两个，而且是同一个field里有两个颜色，能否帮我解决一下please？
fields→sciif,sciUpTop,sci,ssci,eii,pku,cssci,university
Map→SCIIF=IF,sciUpTop=,EI检索=EI,北大中文核心=北核,CSSCI=南核,university=

**Comment by SYL66** [comment]
软件zotero8.0.3，插件版本5.9.7。
已经开启了 PDF Style功能，但是在PDF的阅读界面，背景色只能设置米色、✨和🌙，不能修改其他颜色，也无法重置为白色

**Reply by SYL66** [reply]
发现了，不是插件的问题

**Comment by zkongko027-afk** [comment]
为什么勾选了知网引用数列但没有任何显示呀
